packages/graphql/src/translate/queryAST/ast/filters/authorization-filters/JWTFilter.ts

```diff
@@ -29,35 +29,58 @@ export class JWTFilter extends Filter {
     protected operator: FilterOperator;
     protected JWTClaim: Cypher.Property;
     protected comparisonValue: unknown;
+    protected isNot: boolean;
 
     constructor({
         operator,
         JWTClaim,
         comparisonValue,
+        isNot,
     }: {
         operator: FilterOperator;
         JWTClaim: Cypher.Property;
         comparisonValue: unknown;
+        isNot: boolean;
     }) {
         super();
         this.operator = operator;
         this.JWTClaim = JWTClaim;
         this.comparisonValue = comparisonValue;
+        this.isNot = isNot;
     }
 
     public getChildren(): QueryASTNode[] {
         return [];
     }
 
     public getPredicate(_context: QueryASTContext): Predicate | undefined {
-        return createComparisonOperation({
+        if (this.comparisonValue === null) {
```

Comment:
Is this code path tested?

Reply:
You're right, I've actually removed it because I think it only applies to node properties.